src/ruby/yast/core_ext.rb

```diff
@@ -0,0 +1 @@
+require_relative "core_ext/string"
```

Comment:
I don't think we have to worry about different core extensions creating the same method. The set of extensions will be always kept under control, I don't expect we are going to get mad creating a lot of extensions.

I don't fully like the inheritance approach. It makes impossible to combine two different "core extensions".

I'll try to adapt this PR to use ruby refinements. After considering the options it still look like the way to go for me.